js/catalog-search.js

```diff
@@ -18,9 +18,12 @@ bcpl.catalogSearch = (($, queryStringer, waitForExistence, constants) => {
 	};
 
 	const init = () => {
-		waitForExistence(showResults, () => {
-			$(showResults).prepend(`Showing results for ${getSearchTerm()} <br>`);
-			$(resultsInfoContainerSelector).find('td').first().after(searchCatalogButton);
+		waitForExistence(showResultsSelector, () => {
```

Comment:
you should be able to wait for existence of `resultsInfoContainerSelector` now, or did that not work?

Reply:
Removed un-needed selector and changed wait to the only selector we need.